docs/index.rst

```diff
@@ -0,0 +1,32 @@
+.. pbench documentation master file, created by
+   sphinx-quickstart on Fri Nov  6 22:27:57 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Pbench
+########
+
+A Benchmarking and Performance Analysis Framework
+
+The code base includes three sub-systems. The first is the collection agent, ``pbench-agent``, responsible for providing commands 
+for running benchmarks across one or more systems, while properly collecting the configuration data for those systems, and 
+specified telemetry or data from various tools (``sar``, ``vmstat``, ``perf``, etc.).
+
+The second sub-system is the ``pbench-server``, which is responsible for archiving result tar balls, indexing them, and unpacking 
+them for display.
+
+The third sub-system is the ``web-server`` JS and CSS files, used to display various graphs and results, and any other content 
```

Comment:
Can you adjust the three paragraphs to be the same as published on the land page in the three "cards" entitled "Pbench Agent", "Pbench Server", and "Dashboard"?

Reply:
As per discussion, I have made the requested changes. The three paragraphs in the form of cards will look as :
![One_column](https://user-images.githubusercontent.com/17689823/99608832-b481bf00-2a34-11eb-9cf9-98b629587c7b.png)

Reply:
Layout wise it looks good.  Can you update the text to make the three cards on the main [documentation page](https://distributed-system-analysis.github.io/pbench/)?

Reply:
I have made the requested changes. 

![Screenshot from 2020-11-20 23-46-06](https://user-images.githubusercontent.com/17689823/99835440-9c23b880-2b8a-11eb-99ab-5628e567a657.png)